core/src/main/scala/org/apache/spark/util/JsonProtocol.scala

```diff
@@ -407,6 +407,8 @@ private[spark] object JsonProtocol {
     g.writeFieldName("Accumulables")
     accumulablesToJson(stageInfo.accumulables.values, g)
     g.writeNumberField("Resource Profile Id", stageInfo.resourceProfileId)
+    g.writeBooleanField("Push based shuffle enabled", stageInfo.isPushBasedShuffleEnabled)
+    g.writeNumberField("Shuffle push mergers count", stageInfo.shuffleMergerCount)
```

Comment:
Nit: Capitalization for the first char of each word, following the same style as others shown above